test/pseudo-tty/test-assert-no-color.js

```diff
@@ -17,3 +17,26 @@ assert.throws(
       '-   foo: \'bar\'\n' +
       '- }\n',
   });
+
+{
+  // TODO(puskin94): remove the emitWarning override once the partialDeepStrictEqual method is not experimental anymore
+  // Suppress warnings, necessary otherwise the tools/pseudo-tty.py runner will fail
+  const originalEmitWarning = process.emitWarning;
+  process.emitWarning = () => {};
```

Comment:
```suggestion
  // Suppress warnings, necessary otherwise the tools/pseudo-tty.py runner will fail
  const originalEmitWarning = process.emitWarning;
  process.emitWarning = common.mustCall(() => {});
```

That way the test would fail when it is not called anymore when removing the warning.

The same in the other files.